aws/database/rds/vpc.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  vpc_cidr_block = "10.0.0.0/24"
+}
+
+resource "aws_vpc" "main" {
+  cidr_block           = local.vpc_cidr_block
+  enable_dns_support   = var.publicly_accessible # if DB instance is publicly accessible must be enabled
+  enable_dns_hostnames = var.publicly_accessible # if DB instance is publicly accessible must be enabled
+}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_subnet" "instances_subnets" {
+  count             = 2
+  vpc_id            = aws_vpc.main.id
+  cidr_block        = cidrsubnet(local.vpc_cidr_block, 4, count.index)
```

Comment:
I had problems during dev deploying both subnets to the same cidr block (tf was throwing an error cannot recall which one though), currently running a test for this, deployed, then changed this, and trying to run an apply to change/update this, and at least as of now, it seems like tf is stuck, and has been trying to destroy some VPC stuff for the past 20 min. So I don't think this change actually works. Might be because of the order of my steps, so I can try to run another test once tf is "unstuck" with destroying the infrastructure completely, and then building it new, but tbh that is exactly what caused problems during dev already.

Reply:
can confirm my experience from dev, this doesn't work, because both subnets need their own CIDR block range:
<img width="892" alt="image" src="https://github.com/THEY-Consulting/they-terraform/assets/25197130/c553c4c1-01a9-407b-a20b-e5ee62165eef">

Reply:
yeah, sorry, it was to late. Did misunderstand the logic.